Libraries/LibWeb/HTML/Parser/HTMLParser.cpp

```diff
@@ -2567,15 +2567,13 @@ void HTMLParser::handle_in_body(HTMLToken& token)
         // 1. Insert an HTML element for the token.
         (void)insert_html_element(token);
 
-        // FIXME: 2. If the next token is a U+000A LINE FEED (LF) character token, then ignore that token and move on to the next one. (Newlines at the start of textarea elements are ignored as an authoring convenience.)
```

Comment:
I think we should leave this spec comment here, and maybe note that the statement above is the implementation of that, otherwise step 2 would seem inconspicuously missing

Reply:
Good point!